### ISSUE TITLE
Send SMS without opening default messaging app

### ISSUE THREAD

**Issue by mammadkoma** [dotnet/maui#18375]
### Description

I would like to send SMS by C# code without opening default messaging app.

### Public API Changes

ISms

### Intended Use-Case

Sending SMS in backgroud in single mode and multi sms with try catch and get count of sending

**Comment by mattleibow** [comment]
I do not believe this is possible on iOS and I think for Android you have to mark the app as a default SMS app.

For this it may be better to use a SMS service and use their REST APIs to send messages.

**Comment by mammadkoma** [comment]
I did it in flutter and sms sent correctly :
https://mwaseemaw.medium.com/flutter-send-sms-in-the-background-489672d139f8

**Comment by mattleibow** [comment]
That is just Android I think? This is the issue, only android supports it.

**Comment by jozif001** [comment]
yes it can be done only on Android, and you need to specify the right permission.
we have been using this plugin for Xamarin: https://github.com/cjlotz/Xamarin.Plugins
https://github.com/cjlotz/Xamarin.Plugins/blob/327386dc20e502e6e75503d0dcd7eddb6b4e8be3/Messaging/Plugin.Messaging.Abstractions/ISmsTask.cs#L46

and it works fine, but now with .NET Maui we don't have an alternative.
